### PR TITLE
add Presence

### PR DESCRIPTION
**Pull request by WaldperlachFabi**
## Description 

Added Presence for the Website https://WaldperlachFabi.de/

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

unknown
unknown1

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
Where are the screenshots?
Follow the [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

File: websites/W/WaldperlachFabi/dist/presence.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        presenceData.details = "Viewing contact page";
+    else if (document.location.pathname === "/wp-content/")
+        presenceData.details = "Viewing an video/image";
+    else if (document.location.pathname === "/about/")
+        presenceData.details = "Viewing about page";
+    if (presenceData.details)
+        presence.setActivity(presenceData);
+    else
+        presence.setActivity();
+});
```
Comment (DarkVillager) [inline websites/W/WaldperlachFabi/dist/presence.js:47]: Don't include this file.

File: websites/W/WaldperlachFabi/presence.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        else if (document.location.pathname === "/contact/")
+        presenceData.details = "Viewing contact page";
+        else if (document.location.pathname === "/wp-content/")
+        presenceData.details = "Viewing an video/image";
+        else if (document.location.pathname === "/about/")
+        presenceData.details = "Viewing about page";
+        
+        if (presenceData.details) presence.setActivity(presenceData);
+        else presence.setActivity();
+    });
```
Comment (DarkVillager) [inline websites/W/WaldperlachFabi/presence.ts:49]: Low quality.

Try using a switch statement & variables.

**Review by DarkVillager — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
Low quality presence

**Comment by WaldperlachFabi** [comment]
Sorry here are the Pictures!
image
image
image
image
image

**Pull request closed without merging**